tests/cbmc/stubs/read.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cbmc_proof/nondet.h>
+#include <unistd.h>
+
+static ssize_t round = MAX_BLOB_SIZE;
```

Comment:
This should be its own macro. We don't want to force proof writers to tie it to the blob-size.

Reply:
https://github.com/awslabs/s2n/pull/2071#discussion_r457732766